src/Sarif.Sdk.sln

```diff
@@ -47,7 +47,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{90E0CA8D-0D6
 		BinSkimConfiguration.xml = BinSkimConfiguration.xml
 		build.props = build.props
 		CommonAssemblyInfo.cs = CommonAssemblyInfo.cs
-		NuGet.Config = NuGet.Config
+		..\NuGet.Config = ..\NuGet.Config
```

Comment:
>NuGet.Config [](http://example.com/codeflow?start=23&length=12)

please move this to the .nuget directory

Reply:
Nuget.config is base on a system level Nuget.config,
and then override with the parent folder Nuget.config if any,
and then override with the current folder Nuget.config if any.

the location matters, we can not move to .nuget directory, if so it will only apply to that sub folder.